Makefile

```diff
@@ -29,14 +29,19 @@ build: ## Build/compile the OpenFGA service
 run: build ## Run the OpenFGA server with in-memory storage
 	./openfga run
 
+
+.PHONY: start-postgres-container
+start-postgres-container:
+	docker run -d --name postgres -p 5432:5432 -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=password postgres:14
```

Comment:
can we add an echo here that shows the connection string?

Reply:
Okay. Now it will look like:
```
docker run -d --name postgres -p 5432:5432 -e POSTGRES_USER=postgres -e POSTGRES_PASSWORD=password postgres:14
6200b45f0ad4ec464490becb91faa18c1c8dcf5dc6a94e33c3bae06b642f4897
> use 'postgres://postgres:password@localhost:5432/postgres' to connect to postgres
```